config/routes.rb

```diff
@@ -14,12 +14,15 @@
   resources :lessons, defaults: { format: :json }
   root 'static_pages#home'
   get '/about', to: 'static_pages#about'
-  get '/check_username', to: 'users#check_username', defaults: { format: :json }
-  get '/check_email', to: 'users#check_email', defaults: { format: :json }
   resources :lessons do
     resources :comments
   end
 
+  scope module: 'public' do
+    get '/check_username', to: 'users#check_username', defaults: { format: :json }
```

Comment:
Please, remove here for  '/check_username', '/check_email' defaults: { format: :json } as it works without defaults

Reply:
Done